src/chocolatey/infrastructure.app/services/ChocolateyPackageService.cs

```diff
@@ -420,6 +420,23 @@ public void handle_package_result(PackageResult packageResult, ChocolateyConfigu
 
             remove_pending(packageResult, config);
 
+            // Test to see if exit code is either:
+            // 1641 - restart initiated
+            // 3010 - restart required
+            var rebootExitCodes = new List<int> { 1641, 3010 };

```

Comment:
Minor nitpick on memory/perf - we'd create this list every single time this method is called. Can we move this out to a class level private variable so it is only created once?